Stylesheets/PageStyles/wishlist.css

```diff
@@ -0,0 +1,99 @@
+.main {
+    grid-template-columns: auto;   
+}
+
+.background {
+    grid-template-columns: repeat(2, 1fr);
+    filter: blur(3px);
+    position: fixed;
+    z-index: 0;
+    top: 0;
+}
+
+.product-houses {
+    width: 50vw;
+    height: 50vh;
+}
+
+.product-gryffindor {
+    background-color: #800000;  
+}
+
+.product-slytherin {
+    background-color: #294936;
+}
+
+.product-ravenclaw {
+    background-color: #015783;
+}
+
+.product-hufflepuff {
+    background-color: #FFD63B;
+}
+
+.product {
+    flex-direction: column
+}
+
+.wishlist-heading {
+    display: flex;
+    justify-content: center;
+    z-index: 5;
+    margin-top: 0.5rem;
+    color: var(--st-white);
+    font-family: var(--st-header-text);
+    margin-bottom: 1rem;
+}
+
+.wishlist-div {
+    flex-wrap: wrap;
+    gap: 1rem;
+    margin: 0 4rem;
+    margin-bottom: 2rem;
+}
+
+.product-card {
+    background-color: var(--st-background-color);
+    grid-template-columns: 14rem;
+    grid-template-rows: repeat(6, auto);
+    max-width: 14rem;
+    border-radius: 0.5rem;
+    position: relative;
+}
+
+.product-image, .btn-wishlist {
+    grid-area: 1/1/2/2;
+}
+
+.btn-wishlist {
+    color: var(--st-red);
+}
+
+.product-image {
+    background-color: var(--st-background-color);
+    margin: 0  auto;
+    width: 15rem;
+    height: 17rem;
+    border-top-left-radius: 0.5rem;
+    border-top-right-radius: 0.5rem;
```

Comment:
You can use border-radius: 0.5rem 0  0  0.5rem here.

Reply:
I always get confused in this shorthand property so kept it that way.